CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Send client utc offset with attendee JOIN frame over signalling channel. 
-- Add an unified interface (videofx processor) with new implementation for ML features, namely background blur 2.0 and background replacement 2.0. This includes improvement to the underlying model for better segmentation, better image processing algorithms to blur the background, and other miscellanies bug-fixes/features.
```

Comment:
nit: Do not change the entries from the previous version. Since you will merge this PR after publishing 3.13 to NPM, add your entry under `[3.14.0]`.

Reply:
Sure we will remove this change. Thanks